ocaml/xapi/create_misc.ml

```diff
@@ -569,6 +569,10 @@ let make_software_version ~__context host_info =
   @ v6_version
   @ [
       ("xapi", get_xapi_verstring ())
+    ; ( "xapi_release"
```

Comment:
This isn't the release number in the RPM sense, but a version number. Perhaps name it `toolstack`?

Also, perhaps just use `Build_info.version` here directly rather than unpacking it and putting it back together?